Gemfile

```diff
@@ -1,7 +1,8 @@
 source "https://rubygems.org"
 
 gem "rails", ">= 4.0.0"
-gem "kaminari"
+# FIXME: Fix garage to work with kaminari v1.
+gem "kaminari", "< 1"
```

Comment:
Ah, I'mt getting it. The "pagination" is an option in garage gem, so we can leave this as it is. What do you think of it?